Extensions/pokes.css

```diff
@@ -115,7 +115,6 @@
 	content: "";
```

Comment:
As I said, I did not know what the unicode is, I copied it from tumblr's stylesheet